### PR TITLE
Migrated information and music commands to slash command

### PR DESCRIPTION
**Pull request by RealYusufIsmail**
# PR Title
### Checklist

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines](/CONTRIBUTING.md).
- [x] I agree with the [license](/LICENSE).

### Changes
- [ ] Refactoring
- [x] Internal code
- [ ] External Code / Libraries
- [ ] Documentation
- [ ] Other: \_____

Closes Issue: -

## Description
I have migrated the information and music commands to slash commands meaning they will no longer use the prefix system.

### CODE REVIEW

File: src/main/java/com/github/black0nion/blackonionbot/commands/information/GuildInfoCommand.java
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/commands/information/GuildInfoCommand.java:32]: I recently changed the way to format the date; please use `guild.getTimeCreated().format(BotInformation.DATE_TIME_FORMATTER)`

File: src/main/java/com/github/black0nion/blackonionbot/commands/information/UserInfoCommand.java
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/commands/information/UserInfoCommand.java:62]: You can just use the passed `author` variable

File: src/main/java/com/github/black0nion/blackonionbot/commands/music/QueueCommand.java
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/commands/music/QueueCommand.java:47]: Legacy code by me, you can use `cmde.reply` iirc

**Review by SIMULATAN — CHANGES_REQUESTED** [review]
Found some minor issues, but the rest looks really good, thanks a lot!

**Comment by SIMULATAN** [comment]
Looks good, I'll merge it when I'm on my PC

File: src/main/java/com/github/black0nion/blackonionbot/commands/music/VolumeCommand.java
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/commands/music/VolumeCommand.java:28]: Missing space

**Review by SIMULATAN — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/black0nion/blackonionbot/commands/music/VolumeCommand.java
Comment (RealYusufIsmail) [reply src/main/java/com/github/black0nion/blackonionbot/commands/music/VolumeCommand.java:28]: ah i see

**Comment by RealYusufIsmail** [comment]
> Looks good, I'll merge it when I'm on my PC

I will merge as i have got your approval.

**Pull request merged**
